zrml/neo-swaps/src/types/decision_market_oracle.rs

```diff
@@ -22,6 +22,10 @@ use scale_info::TypeInfo;
 use sp_runtime::DispatchError;
 use zeitgeist_primitives::traits::FutarchyOracle;
 
+/// Struct that implements `FutarchyOracle` using price measurements from liquidity pools.
+///
+/// The oracle evaluates to `true` if and only if the `positive_outcome` is more valuable than the
+/// `negative_outcome` in the liquidity pool specified by `pool_id`.
```

Comment:
Will be changed by audit fixes. It will then use a scoreboard.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1391/commits/346f5836f9c1b348007bb3022824983bc396d556